app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in = command.split(" ")[1]
+    file_out = command.split(" ")[2]
```

Comment:
Make sure you check that the cp input command is correct.

Made changes according to this checklist item
And you can use unpacking to split command parts